CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ Main (unreleased)
   cluster instance to another could have a staleness marker inserted and result
   in a gap in metrics (@thampiotr)
 
+### Other changes
+
+- `prometheus.exporter.snmp`: Updating SNMP exporter from v0.24.1 to v0.26.0.
```

Comment:
maybe you could list the most notable changes

Reply:
I agree with you, but there are too many of them and I'm not sure what to list. I'll just let people reference the exporter's changelog 😅